ecl/network/v2/load_balancer_actions/results.go

```diff
@@ -0,0 +1,38 @@
+package load_balancer_actions
+
+import (
+	"github.com/nttcom/eclcloud"
+)
+
+type commonResult struct {
+	eclcloud.Result
+}
+
+// ExtractResetPassword is a function that accepts a result and extracts a result of reset_password.
+func (r commonResult) ExtractResetPassword() (*LoadBalancerActionResetPassword, error) {
+	var s LoadBalancerActionResetPassword
+	err := r.ExtractInto(&s)
+	return &s, err
+}
+
+// RebootResult represents the result of a reboot operation. Call its
+// ExtractErr method to determine if the request succeeded or failed.
+type RebootResult struct {
+	eclcloud.ErrResult
+}
+
+// ResetPasswordResult represents the result of a Reset Password operation. Call its ExtractResetPassword
+// method to interpret it as an action's result.
+type ResetPasswordResult struct {
+	commonResult
+}
+
+// LoadBalancerActionResetPassword represents a detail of a Reset Password operation.
+type LoadBalancerActionResetPassword struct {
```

Comment:
I prefer shorter name like this.
```suggestion
type ResetPassword struct {
```

Reply:
`ResetPassword` : conflicted to the method name using this structure.
  -> changed to `ResetPasswordDetail`

Reply:
Sorry, I did not realize name conflict. So how is `Password` simply?

Reply:
Nice name! Changed as you suggested.